app/views/stations/show.erb

```diff
@@ -3,4 +3,11 @@
 <h3><%= @station.name %></h3>
 <p><%= @station.dock_count %></p>
 <p><%= @station.city.name %></p>
-<p><%= @station.installation_date %></p>
\ No newline at end of file
+<p><%= @station.installation_date %></p>
+<p><%= Trip.rides_started(@station.id) %></p>
```

Comment:
Generally using the class in your view is a sign that there's a method you could create something at the model level. Instead of creating this on Trip, can you create a method on the Station model? Then  you wouldn't need to pass `@station.id` to this method (it would be the id of the station that the method was living on).